docs/deployment/index.md

```diff
@@ -11,20 +11,46 @@ _For more information about advanced configuration, read our documentation [here
 
 ```
 ❯ docker run -d -p 8999:80 -e PROMITOR_AUTH_APPID='<azure-ad-app-id>'   \
-                         -e PROMITOR_AUTH_APPKEY='<azure-ad-app-key>' \
-                         -v C:/Promitor/metrics-declaration.yaml:/config/metrics-declaration.yaml \ 
-                         tomkerkhove/promitor-scraper
+                           -e PROMITOR_AUTH_APPKEY='<azure-ad-app-key>' \
+                           -v C:/Promitor/metrics-declaration.yaml:/config/metrics-declaration.yaml \ 
+                           tomkerkhove/promitor-scraper
```

Comment:
What is the change in white space here?

Reply:
Looked like it had intended to line up with the line above, so purely aesthetic reasons.